### DISCUSSION TITLE
Export BigQuery - Partition - How to filter by _PARTITIONTIME when querying the view raw_latest?

### DISCUSSION THREAD

**Discussion by bruno-schuermans** [firebase/extensions#857] [Q&A]
Hi,
Partitioning is working fine on the table raw_changelog.
But if you are querying from the view raw_latest, there is no _PARTITIONTIME column you can use to only query part of your data.

**Comment by KristofSochan** [comment]
Hi @bruno-schuermans, did you ever find how this can be done? It seems like a huge limitation that you cannot use partitioning on the view that gives the current state of the collection's documents.
